tsconfig.json

```diff
@@ -1,6 +1,28 @@
 {
   "compilerOptions": {
-    "jsx": "preserve"
```

Comment:
what does changing 'preserve' to 'react' do?

Reply:
I don't know but this is what atlaskit does...